python/packages/nisar/workflows/nisar_l0b_dm2_to_dbf.py

```diff
@@ -45,15 +51,32 @@ def copy_swath_except_echo_h5(fid_in, fid_out, swath_path, frq_pol):
     frq_pol : dict
         A dict of all frequency bands as keys and list of TxRx
         polarization as values.
+    tx_keys : sequence of str, optional
+        Sequence of dataset names to be excluded from tx group in `fid_out`.
+    rx_keys : sequence of str, optional
+        Sequence of dataset names to be excluded from rx group in `fid_out`.
+        Echo datasets are always excluded from RX group!
 
     """
+    # TX/RX dataset names to be excluded over all frequency bands and pols!
+    tx_names = []
+    if tx_keys is not None:
+        tx_names.extend(tx_keys)
+    rx_names = []
+    if rx_keys is not None:
+        rx_names.extend(rx_keys)
```

Comment:
```suggestion
    tx_names = [f"vadidSamplesSubSwath{n}" for n in "12345"]
    if tx_keys is not None:
        tx_names.extend(tx_keys)
    rx_names = [f"{tx}{rx}" for tx in "HVLR" for rx in "HV"]
    if rx_keys is not None:
        rx_names.extend(rx_keys)
```
No big deal, but we could simplify some of the logic below by just initializing the exclude lists.